app/client/pages/user/[user_id].js

```diff
@@ -110,7 +152,7 @@ export default function profile(props) {
                     }}
                   >
                     <Avatar
-                      src={post.avatar}
+                      src={personalValues.image}
```

Comment:
`${API_URL}/user/${personalValues.image}`. image url should be in this form. Correct this and make sure that profile photo is visible